score-rce/src/main/java/io/cloudslang/content/remote/services/RunWindowsCommandService.java

```diff
@@ -0,0 +1,53 @@
+package io.cloudslang.content.remote.services;
+
+import io.cloudslang.content.remote.constants.Constants;
+import io.cloudslang.content.remote.constants.Outputs;
+import io.cloudslang.content.remote.utils.RunWindowsCommandInputs;
+import io.cloudslang.content.remote.utils.CommandUtils;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by pasternd on 07/06/2016.
+ */
+public class RunWindowsCommandService {
+
+    RunWindowsCommandHelper runWindowsCommandHelper = new RunWindowsCommandHelper();
+
+    public Map<String, String> execute(RunWindowsCommandInputs runWindowsCommandInputs) {
+        String command_result;
+        String error_message;
+
+        Map<String, String> results = new HashMap<>();
+
+        try {
+            command_result = launchCmdWinCommand(runWindowsCommandInputs);
+            results.put(Outputs.COMMAND_RESULT, command_result);
+            results.put(Outputs.ERROR_MESSAGE, Constants.EMPTY_STR);
+        } catch (Exception e) {
+            error_message = Constants.INVALID_PARAMETERS;
```

Comment:
I think the error message can be more specific here. "Authentication failed. Invalid username or password", "Host cannot be reached" or "Invalid command at line 2 row 7"  would help the user to identify easier the invalid parameter.